hs_core/models.py

```diff
@@ -1447,55 +1454,46 @@ def scimeta_url(cls, resource_id):
         scimeta_url = reverse('rest_download', kwargs={'path': scimeta_path})
         return scimeta_url
 
+    # TODO: there are too many ways to get to the resourcemap.
+    # 1. {id}/data/resourcemap.xml
+    # 2. {id}/resmap
+    # Choose one!
     @classmethod
     def resmap_url(cls, resource_id):
         resmap_path = "{resource_id}/data/resourcemap.xml".format(resource_id=resource_id)
         resmap_url = reverse('rest_download', kwargs={'path': resmap_path})
         return resmap_url
 
+    # TODO: this is inaccurate; resourcemap.xml != systemmetadata.xml
     @classmethod
     def sysmeta_path(cls, resource_id):
         return "{resource_id}/data/resourcemap.xml".format(resource_id=resource_id)
```

Comment:
@alvacouch If you referring to the ```sysmeta_path()``` then that is not part of the REST endpoint for getting system metadata.  See the following view function that serves the system metadata endpoint:
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_rest_api.py#L531